apps/graphql/src/apps/itinerary/types/inputs/ItinerariesCommonTypesInput.js

```diff
@@ -0,0 +1,112 @@
+// @flow
+
+import {
+  GraphQLInputObjectType,
+  GraphQLEnumType,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLString,
+} from 'graphql';
+import { GraphQLDate } from 'graphql-iso-date';
+
+import PassengersInput from './PassengersInput';
+
+// Define the common GraphQL types accross the Search Queries schema
```

Comment:
Please don't create useless comments, I think it's obvious that you are creating GraphQL types here :)

Reply:
Done